crates/cxx-qt-build/src/lib.rs

```diff
@@ -738,11 +788,21 @@ impl CxxQtBuilder {
                 self.cc_builder.file(files.plain_cpp);
                 if let (Some(qobject), Some(qobject_header)) = (files.qobject, files.qobject_header)
                 {
+                    // Ensure that the generated QObject header is in the include path
+                    // so that qmltyperegistar can include them later
+                    if let Some(dir) = qobject_header.parent() {
+                        self.cc_builder.include(dir);
```

Comment:
Hm, I have two concerns with this:

1.: This should only be done once per QML module, not for every file.

2.: More general: We'll end up adding every qml modules root folder as an include directory.
This sounds like it could lead to conflicts as soon as two QML modules have a file by the same name.
Can we fix this somehow by using a dedicated cc_builder? I believe it's not possible to only add the include path when building the qmltyperegistrar .cpp, right?